pepper-apis/src/test/java/org/broadinstitute/ddp/filter/WhitelistFilterTest.java

```diff
@@ -51,6 +52,7 @@ static void startServer() {
 
         static void stopServer() {
             stop();
+            awaitStop();
```

Comment:
I ran into some race conditions locally in the rate limiting test, so I applied the `awaitStop()` here as well just in case.